docs/tutorials/chsh_with_estimator.ipynb

```diff
@@ -2,30 +2,22 @@
  "cells": [
   {
    "cell_type": "markdown",
-   "id": "bf7f865a",
+   "id": "5dc3c063",
    "metadata": {},
    "source": [
     "<!-- vale IBMQuantum.Definitions = NO -->\n",
     "\n",
-    "# Demonstrate the violation of the CHSH inequality with the Estimator primitive"
```

Comment:
Should this page and the user-transpiled-circuits page should have a top-level `h2` (just called "introduction" or something)? I think otherwise the user always gets scrolled down past the top paragraph.

Reply:
Added!